pkg/engine/states/remote/http/http_state.go

```diff
@@ -20,16 +20,15 @@ import (
 // formats MUST contain 4 "%s" placeholders for tenant, project and stack, since we will replace this format with fmt.Sprintf()
 // Let's get applyURLFormat as an example to demonstrate how this suffix format works.
 //
+//	 Example:
 //
-//  Example:
-//
-//	urlPrefix = "http://kusionstack.io"
-//	applyURLFormat = "/apis/v1/tenants/%s/projects/%s/stacks/%s/clusters/%s/states/"
-//	tenant = "t"
-//	project = "p"
-//	stack = "s"
-//  cluster = "c"
-//	the final request URL = "http://kusionstack.io/apis/v1/tenants/t/projects/p/stacks/s/clusters/c/states"
+//		urlPrefix = "http://kusionstack.io"
+//		applyURLFormat = "/apis/v1/tenants/%s/projects/%s/stacks/%s/clusters/%s/states/"
+//		tenant = "t"
+//		project = "p"
+//		stack = "s"
+//	 cluster = "c"
```

Comment:
the original doc format is bad, see the details here: https://pkg.go.dev/kusionstack.io/kusion@v0.7.2/pkg/engine/states/remote/http#HTTPState;
open a new PR to format all?

Reply:
Yes, the old version golangci-lint can't find this issue, same as the other format issues.